torch/testing/_internal/common_methods_invocations.py

```diff
@@ -14293,15 +14293,6 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
                     promotes_int_to_float=True,
                     supports_out=True,
                     supports_rhs_python_scalar=False,
-                    skips=(
-                        # RuntimeError: mul(): functions with out=... arguments don't support
-                        # automatic differentiation, but one of the arguments requires grad
-                        # https://github.com/pytorch/pytorch/issues/68966
-                        DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_variant_consistency_eager'),
-                        DecorateInfo(unittest.expectedFailure, 'TestMathBits', 'test_conj_view'),
```

Comment:
Probably makes sense to bring these skips back until proper inductor generation is fixed